websites/P/Pulse/presence.ts

```diff
@@ -26,30 +26,12 @@ newStats();
 presence.on("UpdateData", () => {
   const presenceData: PresenceData = {
     largeImageKey: "logo",
-    startTimestamp: browsingStamp
+    smallImageKey: "logo"
+    
   };
-  if (document.location.pathname.startsWith("/lite")) {
-    play = document.querySelector(
-      "body > div > div > div > div > div > div > i"
-    );
-    play = play.className;
-    switch (play) {
-      case "fa fa-play":
-        // Music Paused
-        presenceData.smallImageKey = "pause";
         presenceData.state = (strack || "Loading...") + (" - " + sartist);
         presenceData.details =
           (sdj || "Loading...") + (slisteners || "Loading...") + " Listeners";
-        presenceData.smallImageText = "https://itspulse.net";
-        break;
-      case "fa fa-pause":
-        // Music Playing
-        presenceData.smallImageKey = "play";
-        presenceData.state = (strack || "Loading...") + (" - " + sartist);
-        presenceData.details =
-          (sdj || "Loading...") + (slisteners || "Loading...") + " Listeners";
-        presenceData.smallImageText = "https://itspulse.net";
-    }
-  }
```

Comment:
why did you deleted it?

Reply:
was non-functional and i didn't see the point of it in the presence